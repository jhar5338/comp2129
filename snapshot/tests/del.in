del a
set a 1
get a
del a
get a
bye
