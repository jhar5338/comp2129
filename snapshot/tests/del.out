> no such key

> ok

> [1]

> ok

> no such key

> bye
